frontend/pages/_app.tsx

```diff
@@ -21,7 +21,7 @@ function MyApp({ Component, pageProps }: { Component: React.ComponentProps<any>;
 
         <meta property="og:title" content={pageProps.title || TITLE} />
         <meta property="og:description" content={pageProps.description || DESCRIPTION} />
-        <meta property="og:image" content={pageProps.thumbnail || THUMBNAIL} />
+        <meta property="og:image" content={`https://l2beat.com/og/${pageProps.project || 'overview'}.png`} />
```

Comment:
I think now if you managed to build it we can use vercel env instead of hardcoded url `VERCEL_URL` instead of `https://l2beat.com`